src/app/extensions/organization-hierarchies/shared/hierarchy-switch/hierarchy-switch.component.html

```diff
@@ -16,7 +16,12 @@
       <ng-container *ngIf="groups$ | async as groups">
         <ul *ngIf="groups.length">
           <li *ngFor="let g of groups">
-            <a class="dropdown-item d-block" (click)="groupSelected(g)">{{ g.name }}</a>
+            <a
+              class="dropdown-item d-block"
+              *ngIf="(selectedElement$ | async).id !== g.id"
```

Comment:
we like to have the *ngIf statements before all other properties - change it or leave it as it is, but do it in future
e.g. <a *ngIf="(selectedElement$ | async).id !== g.id" class="dropdown-item d-block" ...